### PR TITLE
Kmp 5

### PR DESCRIPTION
**Pull request by AgnieszkaIwan**

### CODE REVIEW

File: components/SliderPlusDescription.vue [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+};
+</script>
+
+<style scoped>
+.caption {
+  position: absolute;
+  bottom: 20px;
+  left: 20px;
+  background: rgba(0, 0, 0, 0.5);
+  color: white;
```
Comment (PiotrDrechsler) [inline components/SliderPlusDescription.vue]: Use variables for background and color. Using variables improves maintainability, consistency, and readability. It allows for easier updates and ensures a consistent look throughout the application. Anyway, in my opinion the rest of the code is ok :)

**Review by PiotrDrechsler — CHANGES_REQUESTED** [review]

File: components/SliderPlusDescription.vue [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <div class="caption">
+        <h2>{{ slide.caption.header }}</h2>
+        <p>{{ slide.caption.intro }}</p>
+      </div>
+    </v-carousel-item>
+  </v-carousel>
+</template>
+
+<script>
+export default {
```
Comment (sabiku85) [inline components/SliderPlusDescription.vue]: @AgnieszkaIwan could you use <script setup> for consistency with the rest of a project?

File: components/SliderPlusDescription.vue [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-carousel>
+    <v-carousel-item
+      v-for="(slide, index) in slides"
+      :key="index"
+      :src="item.src"
+      cover
```
Comment (sabiku85) [inline components/SliderPlusDescription.vue]: @AgnieszkaIwan according to a clickup's task there should be passed also an alt attribute.

File: components/SliderPlusDescription.vue [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-carousel>
+    <v-carousel-item
+      v-for="(slide, index) in slides"
+      :key="index"
+      :src="item.src"
```
Comment (sabiku85) [inline components/SliderPlusDescription.vue]: @AgnieszkaIwan **slide**.src (not item)?

File: components/SliderPlusDescription.vue [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    },
+  },
+};
+</script>
+
+<style scoped>
+.caption {
+  position: absolute;
+  bottom: 20px;
+  left: 20px;
```
Comment (sabiku85) [inline components/SliderPlusDescription.vue]: @AgnieszkaIwan maybe a "left" value should be more responsive (for example: 50% with appropriate transform: translateX value)?

File: pages/index.vue [hunk 6]
```diff
@@ -69,6 +69,13 @@ const config = useRuntimeConfig();
         <ShortSectionTile imageSrc="" title="" defaultTile />
       </div>
     </section>
+
+    <section>
+    <div>
+    <Slider :slides="slides" />
```
Comment (sabiku85) [inline pages/index.vue]: @AgnieszkaIwan is this code needed right now? Besides, shouldn't this component be imported and named as SliderPlusDescription?

**Review by sabiku85 — CHANGES_REQUESTED** [review]

**Comment by AgnieszkaIwan** [comment]
I have added changes from your comments. I will take care about styling after holiday :)

File: components/SliderPlusDescription.vue [hunk 7]
```diff
@@ -1,9 +1,29 @@
+
+<script setup>
+import { defineProps } from 'vue';
```
Comment (sabiku85) [inline components/SliderPlusDescription.vue]: @AgnieszkaIwan I think this part is redundant - nuxt should auto-import it.

File: components/SliderPlusDescription.vue [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    </v-carousel-item>
+  </v-carousel>
+</template>
+
+
+<style>
+.caption {
+  position: absolute;
+  bottom: 20px;
+  left: 50%;
```
Comment (sabiku85) [inline components/SliderPlusDescription.vue:41]: @AgnieszkaIwan you didn't set a transform: translateX value for this left: 50%, so it probably won't be fully  centered.

File: components/SliderPlusDescription.vue (hunk 7 above)
Comment (AgnieszkaIwan) [reply components/SliderPlusDescription.vue]: I have deleted below part:  
const { slides } = props

File: components/SliderPlusDescription.vue (hunk 8 above)
Comment (AgnieszkaIwan) [reply components/SliderPlusDescription.vue:41]: now should be ok

**Review by AgnieszkaIwan — COMMENTED** [review]
I added some changes in code, could you please check? :)

File: components/SliderPlusDescription.vue (hunk 7 above)
Comment (sabiku85) [reply components/SliderPlusDescription.vue]: @AgnieszkaIwan I meant this part being redundant: `import { defineProps } from 'vue';`

If you'll remove this destructuring: `const { slides } = props`, you aren't able to use "slides" in _v-for_ beneath.

**Review by sabiku85 — CHANGES_REQUESTED** [review]
@AgnieszkaIwan could you get latest changes from _dev_ branch, please? Now you have 12 files in your PR instead of just 2 with your changes only.

File: components/SliderPlusDescription.vue (hunk 7 above)
Comment (AgnieszkaIwan) [reply components/SliderPlusDescription.vue]: should be ok now :)

Thanks!

File: components/SliderPlusDescription.vue (hunk 1 above)
Comment (AgnieszkaIwan) [reply components/SliderPlusDescription.vue]: done, thanks! :)

**Review by AgnieszkaIwan — COMMENTED** [review]
done

File: components/SliderPlusDescription.vue [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+      <div class="caption">
+        <h2>{{ slide.caption.header }}</h2>
+        <p>{{ slide.caption.intro }}</p>
+      </div>
+    </v-carousel-item>
+  </v-carousel>
+</template>
+
+<style>
+@import '../assets/css/styles.css';
```
Comment (sabiku85) [inline components/SliderPlusDescription.vue]: @AgnieszkaIwan sorry, I didn't spot it last time, but why do you import a styles.css file in here? It looks like a duplicate, cause it's already imported by a nuxt.config.ts file.

Comment (AgnieszkaIwan) [reply components/SliderPlusDescription.vue]: done! :)

**Review by sabiku85 — APPROVED** [review]

**Review by PiotrDrechsler — APPROVED** [review]

**Pull request merged**